runtime/server/x86/CMakeLists.txt

```diff
@@ -89,11 +89,13 @@ if(NOT MSVC)
   set(openfst_BINARY_DIR ${fc_base}/openfst-build)
   set(openfst_PREFIX_DIR ${fc_base}/openfst-subbuild/openfst-populate-prefix)
   ExternalProject_Add(openfst
-    URL               https://github.com/mjansche/openfst/archive/1.6.5.zip
-    URL_HASH          SHA256=b720357a464f42e181d7e33f60867b54044007f50baedc8f4458a3926f4a5a78
+    URL               https://github.com/mjansche/openfst/archive/1.6.7.zip
+    URL_HASH          SHA256=6ad450057ca4958de6c09358c66e2a137c5a1cbd2cb365b47866bf905fca72e2
     SOURCE_DIR        ${openfst_SOURCE_DIR}
```

Comment:
thrax 1.2.9 is compilable with openfst 1.6.7, so I upgrade openfst here. Not sure whether it will affect wfst decoding, need further verification.